### PR TITLE
l10nupdate: move to mwdeploy and avoid I/O starvation

### PR DESCRIPTION
**Pull request by RhinosF1**

### CODE REVIEW

File: modules/mediawiki/manifests/cron.pp [hunk 1]
```diff
@@ -1,12 +0,0 @@
-# class: mediawiki::cron
-#
-# Used for CORE crons which should be ran on every MediaWiki server.
-class mediawiki::cron {
-    cron { 'update.php for LocalisationUpdate':
-        ensure  => present,
```
[the remark this answers is not in the document]
Comment (RhinosF1) [reply modules/mediawiki/manifests/cron.pp:6]: I'll do it from the mediawiki profile.

File: modules/mediawiki/manifests/init.pp [hunk 2]
```diff
@@ -158,6 +157,19 @@
             user        => www-data,
             subscribe   => Git::Clone['ErrorPages'],
         }
+        
+        cron { 'l10n-modern-deploy':
```
Comment (paladox) [inline modules/mediawiki/manifests/init.pp:161]: Code style please

File: modules/mediawiki/manifests/init.pp [hunk 3]
```diff
@@ -158,6 +157,19 @@
...
+        command => 'deploy-mediawiki --l10nupdate --servers=${lookup(mediawiki::default_sync)}',
+        user    => 'www-data',
+        minute  => '0',
+        hour    => '23',
+        }
+        
+    }
+    
+    cron {'update.php for LocalisationUpdate':
+        ensure = absent,
```
Comment (Universal-Omega) [inline modules/mediawiki/manifests/init.pp:172]: Should be `ensure  => absent` (missing `>`)

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: modules/mediawiki/manifests/init.pp (hunk 2 above)
Comment (paladox) [reply modules/mediawiki/manifests/init.pp:161]: Resolved.

**Pull request merged**